<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductsForm</class>
 <widget class="QWidget" name="ProductsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Products Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font:12pt;
background-color:rgb(54,54,54);
color: rgb(172, 172, 172);
}
QPushButton{
border-radius:15px;
border:2px solid white;
}
QPushButton:hover{
border:2px solid orange;
}
QPushButton:pressed{
border:3px solid pink;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="WindgetFrame" native="true">
       <property name="minimumSize">
        <size>
         <width>710</width>
         <height>280</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
color: white;
font-weight:bold;
}
</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>645</width>
          <height>245</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="rightMargin">
          <number>15</number>
         </property>
         <property name="bottomMargin">
          <number>15</number>
         </property>
         <property name="spacing">
          <number>15</number>
         </property>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_3" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 115, 172);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="editBtn">
              <property name="minimumSize">
               <size>
                <width>81</width>
                <height>81</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>81</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color:white;</string>
              </property>
              <property name="text">
               <string>Edit Item Details</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="widget_6" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 115, 172);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="addBtn">
              <property name="minimumSize">
               <size>
                <width>81</width>
                <height>81</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>81</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Add New Item</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 115, 172);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="viewBtn">
              <property name="minimumSize">
               <size>
                <width>81</width>
                <height>81</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>81</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>View All Products</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_5" native="true">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>100</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(0, 115, 172);</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="deleteBtn">
              <property name="minimumSize">
               <size>
                <width>81</width>
                <height>81</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>81</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Delete Item</string>
              </property>
              <property name="iconSize">
               <size>
                <width>50</width>
                <height>50</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>94</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>240</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border:0px;
border-radius:1px;
}</string>
     </property>
     <property name="title">
      <string>Tips</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>15</number>
      </property>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="topMargin">
         <number>3</number>
        </property>
        <item row="1" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Displays All products in the database</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Adds New item details to the database</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="viewLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="newLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="1" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Update item details</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Delete an item from database</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="editLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="deleteLabel">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>50</width>
            <height>50</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
